### PR TITLE
Combine Intro-R weeks

### PR DESCRIPTION
**Pull request by brymz**
As per #338.

Weeks are condensed largely by removing detailed R{base} subsetting functions. 'Data Frames 2' has been removed from the course material, with relevant content moved to the remaining `dplyr` lessons. @ethanwhite, have a look to make sure you like the structure and then I will go ahead and update the code and output solutions.

### CODE REVIEW

**Comment by brymz** [comment]
This is ready for review and merge.

One thing I noticed in 'Dplyr Using Databases': The last few 'queries' return `tbl_df` that are `<derived table>` and do not return any `nrow` value. What's more, I couldn't figure any way to interact with the values in these tables. Probably not a big issue here as the exercise questions just ask for tables, but students might stumble on if they are messing around. My only explanation is that it is an `RSQLite` issue as I've never run into this before with `dplyr`.

File: exercises/Data-frames-1-R.md [hunk 1]
```diff
@@ -15,14 +15,7 @@ their volumes (`l * w * h`). You could do this using a spreadsheet, but the
 project that you are working on is going to be generating lots of these files so
 you decide to write a program to automate the process.
 
-Download the data, use `read.csv()` to import it into R, and then print out:
+Download the data, use `read.csv()` to import it into R, and then use the `$` operator to print out:
```
Comment (ethanwhite) [inline exercises/Data-frames-1-R.md]: What is the use of `$` for here? I think I was envisioning just printing out the entire data frame.

Comment (ethanwhite) [inline exercises/Data-frames-1-R.md]: Never mind. Coffee kicking in now.

**Comment by ethanwhite** [comment]
Looks great! The only thing I see is the need to deal with the fact that Assignment r-3 is now empty. We should cascade the number moves so that 4 becomes 3, 5 becomes 4, and either 8 disappears or the spatial lesson gets inserted into the appropriate location even though it will be blank temporarily (as an aside, this sort of thing is why I'm still not overly thrilled about using numbers instead of names for assignments, especially since some folks might not want to teach a week on spatial analyses, but we should save that for another issue).

I'm guessing this may have some merge implications, which we can talk about later today.

**Comment by brymz** [comment]
Last week, we discussed merging this before I made further changes to help avoid merge conflicts. Any last minute objections, @ethanwhite?

**Comment by ethanwhite** [comment]
Nope. Just too many things going on. :scream_cat:

**Pull request merged**